client/my-sites/email/email-management/titan-manage-mailboxes/index.jsx

```diff
@@ -134,7 +134,7 @@ class TitanManageMailboxes extends Component {
 
 						<VerticalNavItemEnhanced
 							description={ translate(
-								'Download our Android and iOS apps to access your emails on the go'
+								"Download Titan's Android and iOS apps to access your emails on the go"
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.